README.md

```diff
@@ -156,13 +159,14 @@ InstanceClient {
   instanceURL: 'instance/e70222d1-acfc-4e00-b046-4a3a9481c53b'
 }
 ```
-Sequence is an app template. Once run, it will become live as a new instance. Instance also receives its own ID (GUID). In this case instance Id is:
+
+Sequence is an app template. Once run, it will become live as a new instance. Instance also receives its own ID (GUID). In this case instance ID is:
```

Comment:
```suggestion
Sequence is an app template. Once it is up and running, it will become a new instance. Instance also receives its own ID (GUID). In this case instance ID is:
```